symbolic-symcache/src/transform/mod.rs

```diff
@@ -24,8 +22,6 @@ pub struct File<'s> {
     pub name: Cow<'s, str>,
     /// The optional directory prefix.
     pub directory: Option<Cow<'s, str>>,
-    /// The optional compilation directory prefix.
-    pub comp_dir: Option<Cow<'s, str>>,
```

Comment:
I think we also should keep this. Since the comp_dir/directory are being joined in weird ways, it would be good to maintain the flexibility here.

I know for sure we have these unit tests related to bcsymbolmaps that exhibit this behavior. would be nice to double-check.